test/repositories/formatters/ApprovalProcessMetaFormatterSpec.scala

```diff
@@ -33,7 +33,9 @@ class ApprovalProcessMetaFormatterSpec extends UnitSpec with ApprovalProcessJson
     |    "title" : "This is the title",
     |    "status" : "SubmittedFor2iReview",
     |    "dateSubmitted" : {"$date": 1583193600000},
```

Comment:
You could test these values are correct in the instance of ApprovalProcessMeta created by the formatter

Reply:
Have updated test to reference those fields